src/components/layout/GenericLayout/Menu/index.tsx

```diff
@@ -0,0 +1,62 @@
+import React, { PropsWithChildren } from 'react'
+import styled from 'styled-components'
+
+import { MEDIA } from 'const'
+
+export const NavStyled = styled.ol`
```

Comment:
move files into own folder please

Reply:
Why `ol`? If you want a list, use`ul`at least. Plus I don't see any reason fort his comp to be a list. You override list display and other innate rendering anyway. And list enforces `<li>` as children
Just use div.

Reply:
I think it's an ordered list, where arguably order matters? OL is a valid semantic list. A

with 's work, but arguably an ordered (or unordered) list is more correct in terms of accessibility?

Reply:
@W3stside this is similar to the comment u did in the header PR, I have the same opinion in this case (small components can have small css in the same file), maybe we should discuss it in the other thread. Also I'll add it to the TODO list agree on the procedure on when to make your one file for the styles.